### PR TITLE
Feature/absence manager

### PR DESCRIPTION
**Pull request by FRickReich**

### CODE REVIEW

File: local/App.react.js [hunk 1]
```diff
@@ -105,7 +105,6 @@ const userRelationEditor = (props) => (
     <div>
         {menu(props.router)}
         <UserRelationEditor
-            tenantId='hard001'
             userId='scott.tiger@example.com'
```
Comment (denic) [inline local/App.react.js]: Is this needed?

File: local/App.react.js [hunk 2]
```diff
@@ -126,7 +124,6 @@ const substitutionPeriodUserEditor = (props) => (
     <div>
         {menu(props.router)}
         <SubstitutionPeriodUserEditor
-            tenantId='hard001'
             userId='scott.tiger@example.com'
```
Comment (denic) [inline local/App.react.js]: And this?

File: src/client/api/UserSubstitution.js [hunk 3]
```diff
@@ -7,9 +7,9 @@ class UserSubstitution extends ApiBase
         return this.ajax.post(`/user/api/users/${userId}/substitutionPeriods`).send(data).then(res => res.body).catch(this.getErrorFromResponse);
     }
 
-    deleteUserSubstitution(id)
+    deleteUserSubstitution(userId, id)
```
Comment (denic) [inline src/client/api/UserSubstitution.js]: Order of userId and id should be switched, as the re relevance of parameters should be decreasing

File: local/App.react.js (hunk 1 above)
Comment (FRickReich) [reply local/App.react.js]: i thought we talked about deleting this?

Comment (denic) [reply local/App.react.js]: My comment references the userId.

Comment (FRickReich) [reply local/App.react.js]: ah, thats just for testing, it does not do anything on the component itself.

**Pull request merged**